sys/shell/commands/sc_icmpv6_echo.c

```diff
@@ -161,16 +145,16 @@ int _icmpv6_ping(int argc, char **argv)
 {
     int count = 3, success = 0, remaining, stat_interval = 0, stat_counter = 0;
     size_t payload_len = 4;
-    timex_t delay = { 1, 0 };
+    uint32_t delay = 1 * SEC_IN_MS;
     char *addr_str;
     ipv6_addr_t addr;
     msg_t msg;
     gnrc_netreg_entry_t *ipv6_entry, my_entry = { NULL, ICMPV6_ECHO_REP,
                                                   thread_getpid()
                                                 };
-    timex_t min_rtt = { UINT32_MAX, UINT32_MAX }, max_rtt = { 0, 0 };
-    timex_t sum_rtt = { 0, 0 };
-    timex_t ping_start;
+    uint64_t min_rtt = UINT64_MAX, max_rtt = 0;
```

Comment:
`min_rtt` and `max_rtt` can be both `uint32_t`. No way that a round-trip time can take `>UINT32_MAX` microseconds